checks/libs/wmi/sampler.py

```diff
@@ -263,45 +259,32 @@ def _format_property_values(self, previous, current):
     @contextmanager
     def get_connection(self):
         """
-        Return an existing, available WMI connection or create a new one.
-        Release, i.e. mark as available at exit.
+        Create a new WMI connection
```

Comment:
I guess, it doesn't really make sense anymore to have this method as a `contextmanager`.